schemas/public/onpremises-kfd-v1alpha2.json

```diff
@@ -1835,10 +1662,64 @@
         }
       },
       "then": {
-        "required": [
-          "type",
-          "velero"
-        ]
+        "required": ["type", "velero"]
+      }
+    },
+    "Spec.Distribution.Modules.Dr.EtcdBackup": {
+      "type": "object",
+      "description": "Configuration for the ETCD backup-restore package.",
+      "additionalProperties": false,
+      "properties": {
+        "schedule": {
+          "type": "string",
+          "description": "The cron expression for the `full` backup schedule (default `0 1 * * *`)."
+        },
+        "s3": {
+          "type": "object",
+          "additionalProperties": false,
+          "description": "Configuration for the remote S3 backend, where Etcd backup-restore will (optionally) send its backups.",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "If true, Etcd backup will save snapshots on S3."
```

Comment:
If not true, what happens?

Reply:
It won't enable any backup, at the moment.

Reply:
This was changed to use the `type` field, which can be `pvc`, `s3`, `all` or `none`.